plugins/uefi-capsule/uefi-capsule.quirk

```diff
@@ -30,6 +30,10 @@ Flags = no-coalesce
 [90706264-e399-575b-a9fb-077ead03b9b4]
 Flags = no-coalesce
 
+# Lenovo T14 Gen 2 Battery
```

Comment:
if we're having a quirk anyway for this GUID, perhaps we should also set the name too

Reply:
Ya, I asked @mrhpearson for a long list of GUIDs so we could set the names correctly -- Mark -- any news on that?

Reply:
I'll have to ask the FW team for those - don't have any on hand. But I think this is a great idea.
<Side note - I need to fix my email filters - I almost missed this one>

Reply:
I discussed with the FW team and have an alternate proposal- as getting GUIDs doesn't scope well for future platform updates.
Their internal guidelines for doing FW updates for battery on LVFS documents that they should always add
`<icon type="stock">battery</icon>`
to the firmware.metainfo.xml

Could we use this instead to identify battery FW - at least for Lenovo products?

Reply:
Oh - and wanted to add, while we're at it - battery FW updates will only work if they system is plugged in. Not sure if that's covered somewhere :)

Reply:
> <icon type="stock">battery</icon>

The icon is only used for display, never policy. I think I'd much rather have the battery updates setting:

    <categories>
      <category>X-Battery</category>
    </categories>

...as that sets the icon, fixes the name in the console output and allows us to handle the device in a different way. See below. If you can make that change in your LVFS documents that would be grand, or I can add some fixup code on the LVFS if that's set in metaphorical stone now.

> Could we use this instead to identify battery FW - at least for Lenovo products

Something like https://github.com/fwupd/fwupd/pull/5541 perhaps.

Reply:
> or I can add some fixup code on the LVFS

I did that too :)

Reply:
I've asked them to add the X-Battery category information for battery FW updates. Will update if that's accepted (it seems sensible to me)

For #5541 - that looks good to me as an alternative too :)

Mark

Reply:
FYI - team are adding requirement to add X-battery category to the internal documentation.

They wanted to check if there was a minimum fwupdmgr version that should be used for this to work? Is it available yet or needs to be implemented? 

I'm happy to have a stab at it myself if it's not done but it might take a while to get into - it's silly season right now with new devices landing.